recorder.go

```diff
@@ -38,7 +47,7 @@ func (r *SpanRecorder) RecordSpan(span tracer.RawSpan) {
 }
 
 func (r *SpanRecorder) loop() error {
-	ticker := time.NewTicker(1 * time.Second)
+	ticker := time.NewTicker(r.flushFrequency)
```

Comment:
This means that in non-testing mode, spans in the buffer will wait for up to a minute to be flushed. This cannot happen - we have to flush every second, but only send empty payloads every minute if testing mode is disabled

Reply:
Ok, done